sej-videos.html

```diff
@@ -22,6 +22,29 @@ <h1>Inspirational Videos</h1>
 
  </div>
 </section>
+
+  <div id="poster">
+   <h1> Hadith</h1>
```

Comment:
H1 should also be one as main title. We already have H1 "Inspirational Videos" so all other headings should be H2, H3 etc according to page structure. In this case these can be H2